database/.env.dist

```diff
@@ -4,4 +4,6 @@ DB_USER=root
 DB_PASSWORD=
 DB_DATABASE=gradido_community
 MIGRATIONS_TABLE=migrations
-MIGRATIONS_DIRECTORY=./migrations/
\ No newline at end of file
+MIGRATIONS_DIRECTORY=./migrations/
+
+TYPEORM_SEEDING_FACTORIES=src/factories/**/*{.ts,.js}
```

Comment:
This means we require an `.env` file for seeding to work? Current behaviour is: `.env` is not needed if you are fine with the default values.